javascript/javascript2/week2/hyf-bay/main.js

```diff
@@ -1,17 +1,56 @@
 console.log("Script loaded");
 const products = getAvailableProducts();
 console.log(products);
+let clickCount = 0;
+let clickCountRating = 0;
 const searchInput = document.getElementById("search");
 const maximumPriceInput = document.getElementById("maximumPrice");
+const buttonSortPrice = document.getElementById("price-sort");
+const buttonSortRating = document.getElementById("rating-sort");
+const productsCount = document.getElementById("count-products");
+productsCount.innerText = `${products.length} available`;
 searchInput.addEventListener("input", filterProducts);
-maximumPriceInput.addEventListener("input", filterProducts);
+maximumPriceInput.addEventListener("input", filterProductsByPrice);
+buttonSortPrice.addEventListener("click", sortProductsByPrice);
+buttonSortRating.addEventListener("click", sortProductsByRating);
 function filterProducts(event) {
   const filteredProducts = products.filter((product) =>
     product.name.toLowerCase().includes(event.target.value)
   );
```

Comment:
Isn't it  good idea to use toLowerCase() method with input value as well..
like  product.name.toLowerCase().includes(event.target.value.toLowerCase()).

Reply:
ah good suggestion!